src/auth.js

```diff
@@ -0,0 +1,31 @@
+//
+// DevHub
+//
+// Copyright © 2018 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Jason Leach on 2018-10-02.
+//
+
+import { ImplicitAuthManager } from '@bcgov/common-web-utils';
+
+const config = {
```

Comment:
use env constants?

Reply:
This won't have a server to pull the env vars from. Also this info does not need to be secured so it does not need to go in the env. I'll talk with you and @pangara about how we want to better organize this. Maybe it should go into `constants.js`. If so I'll fix in a separate PR.